Source/Core/DolphinQt2/InDevelopmentWarning.cpp

```diff
@@ -87,12 +90,16 @@ InDevelopmentWarning::InDevelopmentWarning(QWidget* parent)
   connect(btn_run, &QCommandLinkButton::clicked, this, [this](bool) { accept(); });
   connect(std_buttons->button(QDialogButtonBox::Cancel), &QPushButton::clicked, this,
           [this](bool) { reject(); });
+  connect(hide_future, &QCheckBox::toggled, [](bool hide) {
+    QSettings().setValue(QStringLiteral("show_development_warning"), !hide);
```

Comment:
Why is this using QSettings? What's the policy for one vs. the other?

Reply:
I used it in this case because it's a Qt-specific thing, so I prefer using QSettings here instead using SConfig.

Reply:
I'd like a second opinion here -- I'm personally very much against having two ways of doing the same thing, and I'd much prefer if our UI didn't feel like a second project with separate settings, separate directories, etc.

@lioncash care to give your opinion?

Reply:
Just as an example of why these things are annoying to deal with: does QSettings properly respect portable.txt? I would guess that currently it doesn't.

Reply:
It currently doesn't - which doesn't mean it can't. It's also already in use in various places but only for minor, specific settings (saving gamelist sortings, window sizes and locations etc.)

Reply:
IMO, it would be nice if the UI stuff was kept separate from the core settings, and it's one of the things I found really odd since I started contributing to the project (though I don't mean it should be completely disjoint from how we handle our settings, of course. Ideally they'd be separate, but under a common interface that handles portable.txt, etc; how that would be done though, I don't know).

For example, it seems *kind* of out of place for the set of [booleans](https://github.com/dolphin-emu/dolphin/blob/master/Source/Core/Core/ConfigManager.h#L249) determining what to display in the game list, especially when they may not even be used in some frontends (e.g. Android, particularly with regards to the toolbar boolean(s) above the game list ones) to be sitting inside the Core subproject.

I'm not really completely read-up on how our config stuff is done, but ideally it would be nice if the config file itself remained unified, but the ways the options themselves are exposed could be different (which I *think* is possible already, not sure?).